### PR TITLE
Fixing xcontent imports

### PR DESCRIPTION
**Pull request by khushbr**
**Is your feature request related to a problem? Please provide an existing Issue # , or describe.**
Fixing the xcontent imports to handle the [OpenSearch core refactoring change for XContent](https://github.com/opensearch-project/OpenSearch/commit/cb402cb867d05dc6e33a7b19f8d95548cfb3d445)
Issue: https://github.com/opensearch-project/performance-analyzer/issues/406

**Describe the solution you are proposing**
Fixing the XContent Import in code for 2.x

### Check List
- [ y] New functionality includes testing.
  - [ y] All tests pass
- [N/A] New functionality has been documented.
  - [ N/A] New functionality has javadoc added
- [ y] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/performance-analyzer/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by dzane17 — DISMISSED** [review]

**Comment by dzane17** [comment]
Some UTs failing?

File: src/main/java/org/opensearch/performanceanalyzer/http_action/config/PerformanceAnalyzerOverridesClusterConfigAction.java [hunk 1]
```diff
@@ -15,9 +15,9 @@
 import org.apache.logging.log4j.Logger;
 import org.opensearch.client.node.NodeClient;
 import org.opensearch.common.settings.Settings;
-import org.opensearch.common.xcontent.XContentBuilder;
 import org.opensearch.common.xcontent.XContentHelper;
 import org.opensearch.common.xcontent.XContentType;
+import org.opensearch.core.xcontent.XContentBuilder;
```
Comment (kkhatua) [inline src/main/java/org/opensearch/performanceanalyzer/http_action/config/PerformanceAnalyzerOverridesClusterConfigAction.java:20]: Only XContentBuilder moved to `core` and not the others in `common`?

**Review by kkhatua — DISMISSED** [review]

**Comment by khushbr** [comment]
> Some UTs failing?

Thank you for pointing this out, fixed the failing UTs.

File: src/main/java/org/opensearch/performanceanalyzer/http_action/config/PerformanceAnalyzerOverridesClusterConfigAction.java (hunk 1 above)
Comment (khushbr) [reply src/main/java/org/opensearch/performanceanalyzer/http_action/config/PerformanceAnalyzerOverridesClusterConfigAction.java:20]: Based on the commit in core, the current initiative is to only move the xcontent base classes to common, see more detail at https://github.com/opensearch-project/OpenSearch/commit/cb402cb867d05dc6e33a7b19f8d95548cfb3d445

**Review by sgup432 — APPROVED** [review]

**Pull request merged**